chain/trie/trie.go

```diff
@@ -333,21 +333,31 @@ func (t *Trie) Copy() *Trie {
 }
 
 func (t *Trie) TryTraverse() error {
-	return t.traverse(t.root)
+	return t.traverse(t.root, true)
 }
 
-func (t *Trie) traverse(n node) error {
+func (t *Trie) traverse(n node, needPrint bool) error {
```

Comment:
Do we need to keep `needPrint` in release?

Reply:
needPrint can help compare nodes of current state.